accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerProvider.scala

```diff
@@ -58,8 +58,8 @@ class AccumuloLayerProvider extends AttributeStoreProvider
     AccumuloLayerWriter(instance, store, table)
   }
 
-  def valueReader(uri: URI, store: AttributeStore): ValueReader[LayerId] = {
+  def valueReader(uri: URI, store: AttributeStore): ValueReader[LayerId] with OverzoomingValueReader = {
     val instance = AccumuloInstance(uri)
-    new AccumuloValueReader(instance, store)
+    new AccumuloValueReader(instance, store) with OverzoomingValueReader
```

Comment:
My thought was that `AccumuloValueReader` can `extend` `OverzoomingValueReader` always, avoiding this mix-in syntax. The existance of `OverzoomingValueReader` trait and the self-typing there is just to work around the `ID` type parameter in `ValueReader` not being specific enough.